cfgov/v1/jinja2/v1/includes/organisms/filterable-list-results.html

```diff
@@ -0,0 +1,126 @@
+{# ==========================================================================
+
+   Filterable list results
+
+   ==========================================================================
+
+   Create a list of filtered results given:
+
+   page: Wagtail page object that inherits from FilterableListMixin.
+
+   form: Django form object that was used to filter results.
+
+   has_unfiltered_results: Whether the filterable list has anything to filter.
+
+   has_active_filters: Whether the form had any filters applied.
+
+   results_page: Paginated page of results.
+
+   ========================================================================== #}
+
+{% import 'v1/includes/molecules/notification.html' as notification with context %}
+
+<div class="o-filterable-list-results">
+    {%- if not has_unfiltered_results %}
```

Comment:
Could we invert this if/else? ESLint has a rule around [no negated conditions](https://eslint.org/docs/latest/rules/no-negated-condition), with the guidance they're harder to read.